src/main/java/net/clgd/ccemux/plugins/Plugin.java

```diff
@@ -92,17 +92,17 @@ protected final void registerHook(Hook hook) {
 	public abstract Optional<String> getVersion();
 
 	/**
-	 * The author of the plugin. If an empty <code>Optional</code> is returned,
-	 * no author will be shown to end-users.
+	 * The authors of the plugin. If an empty <code>Set</code> is returned,
+	 * no authors will be shown to end-users.
 	 */
-	public abstract Optional<String> getAuthor();
+	public abstract Set<String> getAuthors();
```

Comment:
Just wondering why you're using `List` instead of `Set`? I'd argue authors should have some sort of order. We could probably get away with `Collection` TBH.

Reply:
My main thought was that there should be no duplicate authors, but in a way, this way, you're not prioritising anyone.